test/assertionsSpec.js

```diff
@@ -123,7 +123,7 @@ describe('assertions', function(){
       return Promise.all([
         good1,
         good2,
-        expect(bad).to.be.rejected
+        expect(bad).to.be.rejected()
```

Comment:
I'm getting `TypeError: expect(...).to.be.rejected is not a function` on this... old chai perhaps?